test/auth/features/oidc/broker.go

```diff
@@ -101,22 +113,35 @@ func BrokerSendEventWithOIDCTokenToDLS() *feature.Feature {
 	// Install DLS sink
 	f.Setup("install dead letter sink", eventshub.Install(dls,
 		eventshub.OIDCReceiverAudience(dlsAudience),
-		eventshub.StartReceiver))
-
-	// Install broker with DLS config
-	brokerConfig := append(
-		broker.WithEnvConfig(),
-		delivery.WithDeadLetterSinkFromDestination(&duckv1.Destination{
-			Ref:      service.AsKReference(dls),
-			Audience: &dlsAudience,
-		}),
-	)
-	f.Setup("install broker", broker.Install(brokerName, brokerConfig...))
+		eventshub.StartReceiverTLS))
+
+	f.Setup("install broker", func(ctx context.Context, t feature.T) {
+		brokerConfig := append(broker.WithEnvConfig(),
+			delivery.WithDeadLetterSinkFromDestination(&duckv1.Destination{
+				Ref:      service.AsKReference(dls),
+				Audience: &dlsAudience,
+				//CACerts: eventshub.GetCaCerts(ctx),
```

Comment:
@creydr Question: Do you have any idea on how should I pass the CACert here? With this line I wrote, it is causing the broker creation fail. And I can't find the error message.

Reply:
When you check the created YAML for the broker with the cert, you see we have a wrong indent of the cert. 
Simply add 2 more spaces in `delivery.WithDeadLetterSinkFromDestination()` when the newlines are replaced: 
```
diff --git a/test/rekt/resources/delivery/delivery.go b/test/rekt/resources/delivery/delivery.go
index 8348e5647..626b62c60 100644
--- a/test/rekt/resources/delivery/delivery.go
+++ b/test/rekt/resources/delivery/delivery.go
@@ -90,7 +90,7 @@ func WithDeadLetterSinkFromDestination(dest *duckv1.Destination) manifest.CfgFn
                if dest.CACerts != nil {
                        // This is a multi-line string and should be indented accordingly.
                        // Replace "new line" with "new line + spaces".
-                       dls["CACerts"] = strings.ReplaceAll(*dest.CACerts, "\n", "\n      ")
+                       dls["CACerts"] = strings.ReplaceAll(*dest.CACerts, "\n", "\n        ")
                }
 
                if dest.Audience != nil {
```